build-system/tasks/check-links.js

```diff
@@ -120,10 +120,7 @@ function checkLinksInFile(file) {
       // codepen returns a 503 for these link checks
       {pattern: /https:\/\/codepen.*/},
       // GitHub PRs and Issues can be assumed to exist
-      {
-        pattern:
-          /https:\/\/github.com\/ampproject\/amphtml\/(pull|issue)\/d+.*/,
```

Comment:
Turns out that the problem was due to the `d+`.

Reply:
Thanks!